connectivity/manifests/echo-ingress-l7-http.yaml

```diff
@@ -9,9 +9,11 @@ spec:
     matchLabels:
       kind: echo
   ingress:
-  # Only allow 'other' client to make a GET /public requests.
+  # Only allow 'other' client and world to make a GET /public requests.
   # Allow GET /private' only if a particular HTTP header is set.
   # Disallow L3 traffic for now, flow matcher doesn't yet support L7 drops.
+  - fromEntities:
+    - world
```

Comment:
Watch out! That is allowing everything from `world` clients and not just 8080 with the below HTTP path.

Reply:
I also think it would be best to have a separate policy. Extending this one to allow `world` increases the risk of false negatives. The `world` identity is often our fallback identity when we fail to look up a pod identity. So if that ever happens with the existing test and we've extended the policy like done here, we will miss a failure.

Reply:
Of course, thank you Paul!